misc/git.talon

```diff
@@ -28,6 +28,9 @@ git commit existing: "git commit -a\n"
 git diff (colour|color) words: "git diff --color-words "
 git diff: "git diff "
 git diff cached: "git diff --cached\n"
+git diff staged: "git diff --staged\n"
```

Comment:
Hmm, `git diff --cached` and `--staged` are synonyms, right? Should we just have one command?

Reply:
Oh, TIL. I did not cross-reference `--cached`. The existing one is definitely good enough, then. I am just going to retain my local alias, since it's already ingrained for me. Going to remove this.